python/orca/example/learn/pytorch/super_resolution/super_resolution.py

```diff
@@ -57,19 +57,24 @@
 parser.add_argument('--backend', type=str, default="bigdl",
                     help='The backend of PyTorch Estimator; '
                          'bigdl, torch_distributed and spark are supported.')
-parser.add_argument('--data_dir', type=str, default="./dataset", help='The path of datesets.')                         
+parser.add_argument('--data_dir', type=str, default="./dataset", help='The path of datesets.')
 opt = parser.parse_args()
 
 print(opt)
 
 if opt.cluster_mode == "local":
     init_orca_context()
-elif opt.cluster_mode == "yarn":
-    additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
-    init_orca_context(cluster_mode="yarn-client", cores=4, num_nodes=2,
+elif opt.cluster_mode.startswith("yarn"):
+    if opt.cluster_mode == "yarn-client":
+        additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
+        init_orca_context(cluster_mode="yarn-client", cores=4, num_nodes=2,
+                      additional_archive=additional)
+    elif opt.cluster_mode == "yarn-cluster":
+        additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
+        init_orca_context(cluster_mode="yarn-cluster", cores=4, num_nodes=2,
```

Comment:
any different between client and cluster mode?

Reply:
```
elif opt.cluster_mode.startswith("yarn"):
    additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
    init_orca_context(opt.cluster_mode="yarn-client", cores=4, num_nodes=2,
                      additional_archive=additional)
 ```